qiskit/algorithms/gradients/qfi_result.py

```diff
@@ -0,0 +1,35 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+QFI result class
+"""
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Any
+
+import numpy as np
+
+from qiskit.providers import Options
+
+
+@dataclass(frozen=True)
+class QFIResult:
+    """Result of QFI."""
+
+    qfis: list[np.ndarray]
```

Comment:
Here this returns `qfis`. In the LinComb constructor it has this `estimator: The estimator used to compute the gradients.` where like many places it says `gradients`. Do we want to keep consistent with using qfi or is qfi/gradient interchangable enough?

Reply:
The docstring in the `LinCombQFI` constructor is wrong. I copy-and-pasted from `LinCombEstimatorGradient` and I forgot to change docstrings...